apps/src/studio/levels.js

```diff
@@ -2643,15 +2579,15 @@ levels.hoc2015_blockly_10 = utils.extend(levels.js_hoc2015_event_score_points,
   editCode: false,
```

Comment:
This line needs to change to `js_hoc2015_win_lose` to stay in sync with hoc2015.script